### DISCUSSION TITLE
Legacy Environment Image

### DISCUSSION THREAD

**Discussion by furkandindar** [google/model-viewer#3778] [Q&A] [answered]
Hello dear comunity,

>environment-image
 Controls the environmental reflection of the model. Normally if skybox-image is set, that image will also be used for the environment-image. Use environment-image to only set the reflection without affecting the background. If neither is specified, default neutral lighting will be applied. If 'legacy' is specified without a skybox, then our old default environment is applied instead.

I guess `environment-image=legacy` is not a hdr like "Spruit Sunrise, Aircraft Workshop, Music Hall" from ModelViewer Editor. If this is the case, what is it? What is the default environment applied when it's set to `legacy` ?

**Comment by elalish** [comment, marked as answer]
You can find the hdr version [here.](https://github.com/google/model-viewer/blob/master/packages/shared-assets/environments/lightroom_14b.hdr)

**Reply by furkandindar** [reply]
I don't think lightroom_14b.hdr and legacy give the same results

![image_2022-09-07_13-20-44](https://user-images.githubusercontent.com/33446963/188856256-0cac55d1-7915-41a9-bc2c-f64446e583d4.png)

**Reply by elalish** [reply]
Huh, I guess that must have been an older version? Anyway, the actual scene used for creating that lighting environment is here: https://github.com/google/model-viewer/blob/master/packages/model-viewer/src/three-components/EnvironmentScene.ts

**Reply by furkandindar** [reply]
Thanks for the help, this should work!
